src/components/tests/omni-panel-status-labels.test.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import { createMount } from '@material-ui/core/test-utils';
+
+import OmniPanelStatusLabels, { LabelProps } from '../omni-panel-status-labels';
+
+const mount = createMount();
+
+const getStatusLabelsComponent = (
+  Component: (props: LabelProps) => React.ReactElement,
+  modeText: string,
+  modalLabelClass: string,
+  name: string,
+  hideTextStyleOverride?: string,
+) => {
+  return (
+    <Component
+      modeText={modeText}
+      modalLabelClass={modalLabelClass}
+      name={name}
+      hideTextStyleOverride={hideTextStyleOverride}
+    />
+  );
+};
+
+const buildLabelsWrapper = (
+  Component: (props: LabelProps) => React.ReactElement,
+  modeText: string,
+  modalLabelClass: string,
+  name: string,
+  hideTextStyleOverride?: string,
+) => {
+  return mount(
+    getStatusLabelsComponent(Component, modeText, modalLabelClass, name, hideTextStyleOverride),
+  );
+};
+
+test('Component renders correctly', () => {
+  expect(
+    toJson(buildLabelsWrapper(OmniPanelStatusLabels, 'State', 'mock-style', 'Robot')),
+  ).toMatchSnapshot();
+
+  expect(
+    toJson(
+      buildLabelsWrapper(OmniPanelStatusLabels, 'State', 'mock-style', 'Robot', 'override-style'),
+    ),
+  ).toMatchSnapshot();
+});
+
+test('Component default style is being override when alternative style is provided', () => {
+  const overrideStyle: string = 'override-style';
```

Comment:
nit: I think you can send 'override-style' directly as a parameter, Looks descriptive enough to me.

Reply:
I get what you mean, but I think it is better to assign it to a variable so as to avoid hardcoding values in the following segments should we want to change the value for some reason in the future.

```  
const wrapper = buildLabelsWrapper(
    OmniPanelStatusLabels,
    'State',
    'mock-style',
    'Robot',
    overrideStyle,  <----- here
  );

  const isOverride: boolean = wrapper
    .find(OmniPanelStatusLabels)
    .at(0)
    .find('h6')
    .hasClass(overrideStyle);   <---- and here
```

Reply:
whoops, I didn't see that, you're right =)